### ISSUE TITLE
usar usbguard

### ISSUE THREAD

**Issue by fauno** [piratas-ar/RocketGatitOS#9]
es como el forro para usb pero en software

https://dkopecek.github.io/usbguard/

**Comment by elKaZe** [comment]
Genial, sisi, tiene algun icono para la barra?

**Comment by fauno** [comment]
elKaZe notifications@github.com writes:

> Genial, sisi, tiene algun icono para la barra?

tiene un applet hecho en qt

## 

}(:=

**Comment by elKaZe** [comment]
Ahi probé el applet, pero no me los deja listados

**Comment by fauno** [comment]
estas corriendo el daemon?

## 

http://partidopirata.com.ar

**Comment by elKaZe** [comment]
Si, cuando enchufo aparece el pendrive, me pregunta que acción quiero, y funciona bien. Pero aunque bloquee o permita luego no me aparece en el listado de dispositivos.

**Comment by elKaZe** [comment]
Estoy usando el repo de copr del desarollador y anda regio, con un iconito mas que sexy!
Así que voy a agregar ese que va maaas o menos de la mano con la versión de Github.

**Comment by fauno** [comment]
viene con las traducciones?

## 

:D

**Comment by elKaZe** [comment]
nein, todavía no, es mas creo que no hay manera de seleccionar el idioma fuera del momento de  compilación de la interfaz
